vllm/v1/worker/tpu_model_runner.py

```diff
@@ -973,6 +992,8 @@ def _dummy_run(self, num_tokens: int) -> None:
             num_seqs=num_seqs,
         )
 
+        xm.mark_step()  # Capture tensors created when setting up
```

Comment:
It was actually intended not to have this mark_step as the lines above are just device moves.

Reply:
Oh ok, I put it in earlier to fix a recompilation issue, let me see if it's still there

Reply:
Yep that recompilation is gone